Makefile

```diff
@@ -0,0 +1,15 @@
+.PHONY: build clean run
+
+current_dir := $(shell pwd)
+current_user := $(shell id -u)
+
+build:
+	docker run --rm -v $(current_dir):/opt/accountapp/ -u $(current_user):$(current_user) -w /opt/accountapp --entrypoint ./gradlew java:8 --no-daemon --info war
```

Comment:
I suppose those who don't have ruby locally :)
Joke aside, I just reused the same docker image that the one defined in Jenkinsfile.
So we are sure to have the 'same' container that we one build by Jenkins
So I also update the Jenkinsfile to use openjdk:8-jdk